llpc/test/shaderdb/OpKill_TestFunctionUnreachable_lit.spvasm

```diff
@@ -0,0 +1,86 @@
+; BEGIN_SHADERTEST
+; RUN: amdllpc -spvgen-dir=%spvgendir% -v %gfxip %s | FileCheck -check-prefix=SHADERTEST %s
+
+; SHADERTEST-LABEL: {{^// LLPC.*}} SPIRV-to-LLVM translation results
+; SHADERTEST: unreachable
+
+; SHADERTEST-LABEL: {{^// LLPC.*}} SPIR-V lowering results
+; SHADERTEST: call void{{.*}} @lgc.create.kill
+; SHADERTEST-NEXT: br label
```

Comment:
What does this test? There's very little context around the instructions being matched.